### PR TITLE
Make scp_http_int generic. Add user for bits.gluster.org

### PR DESCRIPTION
**Pull request by nigelbabu**

### CODE REVIEW

**Comment by nigelbabu** [comment]
@mscherer Can you look at this and give me feedback/merge it in? :)

**Comment by mscherer** [comment]
Merged, being pushed

File: roles/scp_http_int/tasks/http_int_credential.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - name: Get the public key from Jenkins server
-  shell: cat /root/ci-logs/ci-logs.pub
+  shell: "{{cat /root/{{ target_name }}/{{ target_name }}.pub}}"
```
Comment (mscherer) [inline roles/scp_http_int/tasks/http_int_credential.yml]: No need to add '{{' at the start of the string. I didn't notice before pushing, sorry (but it broke nothing, so not a big deal). I am fixing it.

**Comment by mscherer** [comment]
Closing as it was merged

**Pull request closed without merging**